packages/cohere/src/convert-to-cohere-chat-prompt.ts

```diff
@@ -71,9 +73,19 @@ export function convertToCohereChatPrompt(
         break;
       }
       case 'tool': {
-        throw new UnsupportedFunctionalityError({
-          functionality: 'tool role',
+        messages.push({
+          role: 'TOOL',
+          tool_results: content.map(toolResult => ({
+            call: {
+              name: toolResult.toolName,
+              // TODO: Populate parameters
+              parameters: {},
```

Comment:
open todo

Reply:
sounds good. imo a note saying that (vs having a todo) might be better here. we don't need to take action, but it would be good to have more context in case we run into this in the future.